Files/MultilingualResources/Files.it-IT.xlf

```diff
@@ -144,7 +144,7 @@
         </trans-unit>
         <trans-unit id="StatusBarControlInvertSelection.Text" translate="yes" xml:space="preserve">
           <source>Invert Selection</source>
-          <target state="new">Invert Selection</target>
+          <target state="new">Inverti selezione</target>
```

Comment:
Please change the state from `new` to `translated`.

Reply:
Got it

Reply:
It looks like the state for "Invert Selection" still needs to be updated.